console/src/components/Services/RemoteSchema/Permissions/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
```

Comment:
1. Keep types close to the component. 
 
2. In this case, we shouldn't "hardcode types", we need to let TypeScript to infer types for component's props. Otherwise, the idea of using TypeScript doesn't make much sense.

Example typing of Connected components:

```ts
const connector = connect(mapPropsToState, (dispatch: Dispatch<ACTION>) => ({
  dispatch,
}));

type InjectedProps = ConnectedProps<typeof connector>;

interface Props extends ExternalProps, InjectedProps {} // this is what you want to use instead of `PermWrapperProps & PermissionsProps`.

const ConnectedComponent = connector(Component);
export default ConnectedComponent;
```